package.json

```diff
@@ -138,33 +146,94 @@
     "web components"
   ],
   "scripts": {
-    "build": "npm-run-all build:sass sass:render",
-    "build:test": "npm-run-all test linters",
-    "build:release": "npm-run-all build build:test build:api build:docs bundler types postinstall",
-    "build:ci": "npm-run-all sweep build:release",
-    "build:api": "wca analyze 'src/auro-form.js' --outFiles docs/api.md",
-    "build:dev:assets": "npm-run-all build:sass:component postCss:component sass:render build:docs",
-    "build:docs": "node scripts/generateDocs.mjs",
-    "build:sass": "npm-run-all build:sass:component postCss:component sass:render",
-    "build:sass:component": "sass --no-source-map src:src",
-    "build:watch": "nodemon -e scss,js,html --watch src --watch apiExamples/**/* --exec npm run build:dev:assets",
+   "dev": "npm run dev:input",
+   "build": "npm-run-all build:input build:dropdown build:sass sass:render types",
+   "build:test": "npm-run-all test linters",
+   "build:release": "npm-run-all build build:test build:api build:docs bundler postinstall",
+   "build:ci": "npm-run-all sweep build:release",
+   "postinstall": "node ./node_modules/@aurodesignsystem/auro-library/scripts/build/postinstall.mjs",
+   "types": "tsc",
+
+   "build:api": "wca analyze src/auro-dropdown.js --outFiles docs/api.md",
```

Comment:
Script probably needs to be updated to handle input as well as dropdown

Reply:
Issue tracked here: https://github.com/AlaskaAirlines/auro-form/issues/12